pages/doc/proxies_preprocessor_rules.md

```diff
@@ -324,14 +324,13 @@ Enables you to pass only metrics that are provided in a `preprocessor_rules.yaml
       cacheSize: 10000
     names:
       - "metrics.1"
-      - "/metrics.2.*/"
+      - "/metrics\\.2.*/"
```

Comment:
@laullon Given your comment https://vmware.slack.com/archives/GUEQ9NMNX/p1665995709630649?thread_ts=1665525940.800769&cid=GUEQ9NMNX, should this be:
` - "/metrics\.2.*/"`

Reply:
Yes it should be `\.`